docker/Dockerfile.gateway

```diff
@@ -0,0 +1,34 @@
+
+ARG spdk_version
+
+FROM spdk:${spdk_version:-latest} as spdk
+
+#FROM fedora:33 AS base
+
+#COPY --from=spdk /tmp/rpms/*.rpm /tmp/
```

Comment:
Why are these lines commented out?  If they are not needed, let's remove them.

Reply:
removed the commented out lines